src/apis/member/api.ts

```diff
@@ -0,0 +1,5 @@
+import type { MemberProfileRes } from './types'
+import { http } from '@utils/http'
+
+export const getMemberProfile = async () =>
```

Comment:
```suggestion
export const getMyProfile = async () =>
```
memberProfile api가 이미 있어서 myProfile이 되어야할 것 같습니다!

Reply:
member profile은 타사용자 프로필 조회일까요 ?

Reply:
[반영 완료 ! ](https://github.com/price-offer/offer-fe/pull/175/commits/e1b19a97ac215f4be7d4b0ecd8e7b06d0c7aa3ed)